.changeset/three-dolphins-float.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+add support for service bindings in `wrangler pages dev`
```

Comment:
Can we flesh this out a little with an example of `--service MY_BINDING=my-script`?

Reply:
sure 🙂👍